src/lib/services.ts

```diff
@@ -0,0 +1,31 @@
+import { ProviderService } from '@superfaceai/ast';
+
+export class ServicesSelector {
```

Comment:
1. can this just be ServiceSelector? ServicesSelector triggers me in unexpected ways
2. can you make an IServiceSelector for this with just the `getUrl` method and let this implement it (and interpreter consume just the interface)?